File: AbaTodayInc.Web/ClientApp/src/app/models/envia-shipping/package.ts
import {Dimensions} from "./dimensions";

export interface Package {
    content: string;
    amount: number;
    type: string;
    weight: number;
    insurance?: number;
    declaredValue?: number;
    weightUnit: string;
    lengthUnit: string;
    dimensions: Dimensions;
    //AbaToday properties
    containerId?: string;
    productId?: string;
}

File: AbaTodayInc.Domain/EnviaShipping/Request/Package.cs
using System;

namespace AbaToday.Domain.EnviaShipping.Request
{
    public class Package
    {
        public string Content { get; set; }
        public int Amount { get; set; }
        public string Type { get; set; }
        public double Weight { get; set; }
        public int Insurance { get; set; }
        public int DeclaredValue { get; set; }
        public string WeightUnit { get; set; }
        public string LengthUnit { get; set; }
        public Dimensions Dimensions { get; set; }
        //AbaToday properties
        public Guid? CartonStockId { get; set; }
        public Guid? ProductId { get; set; }
    }
}